presto-spi/src/test/java/com/facebook/presto/spi/block/ColumnarTestUtils.java

```diff
@@ -0,0 +1,154 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.block;
+
+import io.airlift.slice.DynamicSliceOutput;
+import io.airlift.slice.Slice;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
+
+final class ColumnarTestUtils
```

Comment:
I'm concerned about copying `AbstractTestBlocks`. It contains quite a few hacky code, especially `assertPositionValue`.

For the others, I'm not as concerned. But I think putting these in one place is easier when we need to find things when writing new tests.